docs/reference/remap/functions/encode_logfmt.cue

```diff
@@ -5,6 +5,11 @@ remap: functions: encode_logfmt: {
 	description: #"""
 		Encodes the `value` to [logfmt](\#(urls.logfmt)).
 		"""#
+	notices: [
+		"""
+			
+			""",
+	]
```

Comment:
I assume this was added by accident?

Reply:
It was supposed to be `notices:     functions.encode_key_value.notices`